README.md

```
@@ -1,21 +1,71 @@
-# reto-winter-2025-ithaka-frontend
+# Reto Winter 2025 - Ithaka Frontend
 
-## Getting Started
+## Proposito  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)